docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Create the observations
</pre><p>I think by observations here, you rather mean a discretization of the 2D grid. Since it is likely people might reuse this code for higher dimensional tasks, it might be best to clarify this is purely for plotting purposes, and nothing to do with the core of the algorithm.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>ok</p>